opensearch-operator/pkg/builders/cluster.go

```diff
@@ -185,6 +185,22 @@ func NewSTSForNodePool(
 
 	image := helpers.ResolveImage(cr, &node)
 
+	var mainCommand []string
+	var com string
+	com = "yes | ./bin/opensearch-plugin install"
+	if len(cr.Spec.General.PluginsList) > 0 {
+		mainCommand = append(mainCommand, "/bin/bash", "-c")
+		for index, plugin := range cr.Spec.General.PluginsList {
+			fmt.Println(index, plugin)
+			com = com + " '" + strings.Replace(plugin, "'", "\\'", -1) + "'"
+		}
+
+		com = com + " ; ./opensearch-docker-entrypoint.sh"
```

Comment:
```suggestion
		com = com + " && ./opensearch-docker-entrypoint.sh"
```
Start should only continue if plugin installation is successful